src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/DependencyAnalysis/Target_ARM/ARMReadyToRunGenericHelperNode.cs

```diff
@@ -39,6 +39,18 @@ protected void EmitDictionaryLookup(NodeFactory factory, ref ARMEmitter encoder,
 
             // Load the generic dictionary cell
             encoder.EmitLDR(result, context, dictionarySlot * factory.Target.PointerSize);
+
+            // If there's any invalid entries, we need to test for them
+            //
+            // Only do this in relocsOnly to make it easier to weed out bugs - the _hasInvalidEntries
```

Comment:
I know this is copy paste but the comment is wrong. Could you fix the other 2 instances too please?

```suggestion
            // Skip this in relocsOnly to make it easier to weed out bugs - the _hasInvalidEntries
```

Reply:
Yep, will do.